voter/src/main/java/org/terracotta/voter/ActiveVoter.java

```diff
@@ -60,14 +57,19 @@
 
   private final Thread voter;
   private final String id;
-  private final String[] servers;
 
   private volatile boolean active = false;
+  private volatile ClientVoterManager activeClientVoterManager = null;
+  private volatile List<ClientVoterManager> voterManagers;
```

Comment:
having volatile collections is tough to look at.  Can concurrency be controlled a different way?

Reply:
Will change the voaltile list of voterManagers to final list.